libcaliptra/examples/generic/main.c

```diff
@@ -83,6 +83,10 @@ int main(int argc, char *argv[])
         .image_bundle = read_file_or_exit(FW_PATH),
         .fuses = {{0}},
     };
+    struct caliptra_image_manifest *manifest =
+        (struct caliptra_image_manifest *)info.image_bundle.data;
+    if (manifest->header.flags & 1)
+        info.apb_pauser = manifest->header.pl0_pauser;
```

Comment:
Should this have an else with the default value?
```suggestion
    if (manifest->header.flags & 1) {
        info.apb_pauser = manifest->header.pl0_pauser;
    } else {
        info.apb_pauser = 1;
    }
```

Reply:
If there is no PL0 PAUSER then I think it shouldn't matter what the PAUSER is. The test harness will set up valid pausers with whatever value is passed here.